SoObjects/SOGo/SOGoGCSFolder.m

```diff
@@ -1754,39 +1751,45 @@ - (void) removeAclsForUsers: (NSArray *) users
             forObjectAtPath: (NSArray *) objectPathArray
 {
   EOQualifier *qualifier;
-  NSString *uid, *uids, *qs, *objectPath, *domain;
+  NSString *uid, *uids, *qs, *objectPath;
   NSMutableArray *usersAndGroups, *groupsMembers;
   NSMutableDictionary *aclsForObject;
-  SOGoGroup *group;
+
   unsigned int i;
 
   if ([users count] > 0)
     {
-      domain = [[context activeUser] domain];
       usersAndGroups = [NSMutableArray arrayWithArray: users];
       groupsMembers = [NSMutableArray array];
       for (i = 0; i < [usersAndGroups count]; i++)
         {
+          NSDictionary *dict;
+
           uid = [usersAndGroups objectAtIndex: i];
-          group = [SOGoGroup groupWithIdentifier: uid inDomain: domain];
-          if (group)
+          dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: uid];
```

Comment:
Respect the domain by using `[SOGoUserManager contactInfosForUserWithUIDorEmail:inDomain:]`.

Reply:
Resolved in commit 948c89f